internal/web/app/index.html

```diff
@@ -1459,13 +1459,92 @@ <h3 class="font-semibold text-gray-900 dark:text-white">
+                        ></path></svg
+                      ><span class="sr-only">Show information</span></a
+                    >
+                    <span id="configuration-summary" class="selected-value flex items-center gap-2 ml-auto min-w-0 text-gray-900 dark:text-white"><span class="truncate">none</span></span>
+                    <svg
+                      data-accordion-icon
+                      class="w-3 h-3 shrink-0"
+                      aria-hidden="true"
+                      xmlns="http://www.w3.org/2000/svg"
+                      fill="none"
+                      viewBox="0 0 10 6"
+                    >
+                      <path
+                        stroke="currentColor"
+                        stroke-linecap="round"
+                        stroke-linejoin="round"
+                        stroke-width="2"
+                        d="M9 5 5 1 1 5"
+                      />
+                    </svg>
+                  </button>
+                </h2>
+                <div
+                  data-popover
+                  id="configuration-popover-description"
+                  role="tooltip"
+                  class="absolute z-10 invisible inline-block text-sm text-gray-500 transition-opacity duration-300 bg-white border border-gray-200 rounded-lg shadow-xs opacity-0 w-72 dark:bg-gray-800 dark:border-gray-600 dark:text-gray-400"
+                >
+                  <div class="p-3 space-y-2">
+                    <h3 class="font-semibold text-gray-900 dark:text-white">
+                      What is a cloud-config?
+                    </h3>
+                    <p>
+                      A <code>cloud-config.yaml</code> file allows you to preconfigure your Kairos system with users, network, and more. It is applied at first boot. See the <a href="https://kairos.io/docs/architecture/cloud-init/" class="font-medium text-blue-600 underline dark:text-blue-500 hover:no-underline" target="_blank">Kairos documentation</a> for details and examples.
+                    </p>
+                  </div>
+                  <div data-popper-arrow></div>
+                </div>
+                <div
+                  id="accordion-body-configuration"
+                  class="hidden p-5 border border-b-0 border-gray-200 dark:border-gray-700"
+                  aria-labelledby="accordion-heading-configuration"
+                >
+                  <label for="cloud_config" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">Paste your <code>cloud-config.yaml</code> here (optional):</label>
+                  <textarea
+                    id="cloud_config"
+                    name="cloud_config"
+                    rows="10"
+                    class="font-mono w-full p-4 rounded-lg bg-gray-900 text-green-400 border border-gray-700 focus:ring-blue-500 focus:border-blue-500 resize-vertical shadow-inner"
+                    placeholder="#cloud-config\nusers:\n  - name: kairos\n    ..."
```

Comment:
`\n` looks weird as a placeholder:

![image](https://github.com/user-attachments/assets/46822cd3-f4eb-439f-954e-0f71900129e7)


but I'm not sure we can have real multi-line placeholders. If not, ignore my comment.